src/Uno.UI/UI/Xaml/Controls/ListViewBase/NativeListViewBase.Android.cs

```diff
@@ -127,7 +127,7 @@ public ScrollBarVisibility HorizontalScrollBarVisibility
 		{
 			get
 			{
-				return HorizontalScrollBarEnabled ? ScrollBarVisibility.Visible : ScrollBarVisibility.Hidden;
+				return HorizontalScrollBarEnabled ? ScrollBarVisibility.Visible : ScrollBarVisibility.Disabled;
```

Comment:
@davidjohnoliver I definitively need your input on this one :)

Reply:
tbh I struggle with the nuances of scroll bar enablovisibleness... I don't have a problem with this change, but should we be instead preserving (and returning) the value sent to `set`? Currently we're throwing that information away because the underlying native property doesn't accommodate the distinction.

Reply:
I found that there is a good explanation in the MS docs actually https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.scrollbarvisibility?view=winrt-20348, but the mess starts when you also consider the `ScrollMode` ... each time I've to spend way to much time to remind all possible values, including the weird case where you can have a `ScrollViewer` which can be scrolled only programmatically but displaying scrollbars.

Anyway you're right, I was a bit lazy, but keeping the "real" value in a backing field is much better.